website/docs/r/organization.html.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organization"
+sidebar_current: "docs-resource-tfe-organization-x"
+description: |-
+  Manages organizations.
+---
+
+# tfe_organization
+
+Manages organizations.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_organization" "organization" {
+	name = "my-org-name"
```

Comment:
_minor_ it'd be nice to fix the spacing here

Reply:
Ah... Sorry about that! I fixed all the examples, but didn't look at the examples in the docs. Fixed all of them now!